lib/Mojo/Template.pm

```diff
@@ -167,6 +168,7 @@ sub process {
     return Mojo::Exception->new($@)->inspect($self->unparsed, $code)
       ->trace->verbose(1)
       unless $compiled = eval $self->_wrap($code, @_);
+    monkey_patch $self->namespace, '_compiled', $compiled;
```

Comment:
This is horrible code. Every time a new template gets compiled, the result is monkey patched into the shared namespace, redefining whatever was there previously. I'm assuming it is meant to assign a name to the otherwise anonymous coderef. That does not actually work reliably, because `monkey_patch` relies for this on `Sub::Util::set_subname`, which is only an optional dependency and not always available.